src/main/resources/application.properties

```diff
@@ -2,4 +2,4 @@ spring.mvc.view.prefix=/WEB-INF/views/
 spring.mvc.view.suffix=.jsp
 spring.datasource.url=jdbc:mysql://localhost:3306/testudo_bank
 spring.datasource.username=root
-spring.datasource.password=<Put MySQL Server Password Here>
\ No newline at end of file
+spring.datasource.password=RayUnico1!
```

Comment:
In the future, don't push credentials info to your branch.

In your local repo, use git stash push <credentials.py file path> to make the password go away while you are doing git add . and git commit on your changes. Then, just do git stash pop when you want that password back.

You can do the same with the application.properties file path.